com.microsoft.mrtk.input/Visualizers/ControllerVisualizer/ControllerVisualizer.cs

```diff
@@ -102,7 +110,7 @@ private void RenderControllerVisuals(UnityInputSystem.InputDevice inputDevice)
                 }
                 else
                 {
-                    useFallbackVisuals = !HandsUtils.GetSubsystem().TryGetJoint(TrackedHandJoint.Palm, handNode, out _);
+                    useFallbackVisuals = !(HandsAggregator != null && HandsAggregator.TryGetJoint(TrackedHandJoint.IndexTip, handNode, out _));
```

Comment:
nit: we could rework some of the `!`s and `()`s out
```suggestion
                    useFallbackVisuals = HandsAggregator == null || !HandsAggregator.TryGetJoint(TrackedHandJoint.IndexTip, handNode, out _);
```